### ISSUE TITLE
Windows default path format causes errors with inspec check

### ISSUE THREAD

**Issue by grdnrio** [inspec/inspec#672]
### Description

Inspec check using autocomplete for paths in Powershell causes an error. Deleting the .<directory>\ dot and slashes works.
### InSpec and Platform Version

Inspec version: 0.19.1
Windows 10 x64 1511
### Replication Case

``` powershell
chef exec inspec check .\cis-ubuntu14.04-level1\
C:/Users/jgardiner/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.1/lib/inspec/profile.rb:27:in `resolve_target': Don't understand inspec profile in ".\\cis-ubuntu14.04-level1\\", it doesn't look like a supported profile structure. (RuntimeError)
        from C:/Users/jgardiner/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.1/lib/inspec/profile.rb:34:in `for_target'
        from C:/Users/jgardiner/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.1/lib/inspec/cli.rb:55:in `check'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
        from C:/Users/jgardiner/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.1/bin/inspec:9:in `<top (required)>'
        from C:/Users/jgardiner/AppData/Local/chefdk/gem/ruby/2.1.0/bin/inspec:22:in `load'
        from C:/Users/jgardiner/AppData/Local/chefdk/gem/ruby/2.1.0/bin/inspec:22:in `<main>'
```
### Possible Solutions

Workaround is to manually remove the backslahes and dot.

**Comment by chris-rock** [comment]
Huh. That is a bumper. We need to fix this. Thanks for reporting @grdnrio

**Comment by mhedgpeth** [comment]
@chris-rock something I could take, perhaps? If so at what level would you change it?

**Comment by chris-rock** [comment]
@mhedgpeth for sure. I try to write down the call history to help identifying the issue:
- lib/inspec/cli.rb: https://github.com/chef/inspec/blob/master/lib/inspec/cli.rb#L113
- lib/utils/base_cli.rb: https://github.com/chef/inspec/blob/master/lib/utils/base_cli.rb#L69
- lib/inspec/runner.rb: https://github.com/chef/inspec/blob/master/lib/inspec/runner.rb#L50
- lib/inspec/profile.rb: https://github.com/chef/inspec/blob/master/lib/inspec/profile.rb#L19
  -lib/fetchers/local.rb : https://github.com/chef/inspec/blob/master/lib/fetchers/local.rb#L13

We need to find out, why the fetcher `local.rb` does not identify the path as a profile.

**Comment by mhedgpeth** [comment]
As I looked at it I thought that adding this line to `fetchers_test.rb` would test it:

```
    # deal with powershell default path created when pressing tab
    %w{.\path\to\b}                => %w{path/to/b}
```

And in local, I would either just explicitly get rid of a ".\" at the start of the target or I would find a ruby-friendly way to understand that path.

Is that the right approach?

**Comment by chris-rock** [comment]
Yeah, that is the right approach. Looks like ruby does not resolve the relative paths properly. Once we know how to tackle it properly we should add that feature to the local fetcher.

**Comment by ksubrama** [comment]
Oh yes.  Ruby and expand_path and it's platform specific behavior is...  hard.  I'll try to poke at it this week.

**Comment by chris-rock** [comment]
should be fixed by #963, please reopen if this continues to be an issue
